src/main.cc

```diff
@@ -564,13 +564,21 @@ int main( int argc, char ** argv )
   QLocale::setDefault( locale );
   QApplication::setLayoutDirection( locale.textDirection() );
 
-  if ( !qtTranslator.load( "qt_extra_" + localeName, Config::getLocDir() ) ) {
-    qtTranslator.load( "qt_extra_" + localeName, QLibraryInfo::location( QLibraryInfo::TranslationsPath ) );
+  if ( qtTranslator.load( "qt_extra_" + localeName, Config::getLocDir() )
+       || qtTranslator.load( "qt_extra_" + localeName, QLibraryInfo::path( QLibraryInfo::TranslationsPath ) ) ) {
```

Comment:
The 2nd condition won't be evaluated if the first one success.

https://en.wikipedia.org/wiki/Short-circuit_evaluation